### PR TITLE
miscellaneous fixes 

### PR DESCRIPTION
**Pull request by osheroff**
@vanchi-zendesk in prepping for kafka, found these little things wrong with maxwell.

### CODE REVIEW

File: src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java [hunk 1]
```diff
@@ -18,21 +20,36 @@
...
+
+	}
+
 	public static List<SchemaChange> parse(String currentDB, String sql) {
-		ANTLRInputStream input = new ANTLRInputStream(sql);
-		mysqlLexer lexer = new mysqlLexer(input);
-		CommonTokenStream tokens = new CommonTokenStream(lexer);
-		mysqlParser parser = new mysqlParser(tokens);
+		if ( SQL_BLACKLIST.contains(sql))
+			return null;
```
Comment (vanchi-zendesk) [inline src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java]: what's this stuff? I thought the binlog stream does not have transactional stuff..? (I mean, only write after it succeeds no?)

File: src/main/java/com/zendesk/maxwell/schema/SchemaStore.java [hunk 2]
```diff
@@ -100,11 +101,38 @@ private void saveSchema() throws SQLException {
...
+		}
+
+		PreparedStatement columnInsert = connection.prepareStatement(insertColumnSQL);
+		int i = 1;
+
+		for (Object o : columnData)
+			columnInsert.setObject(i++,  o);
+
+		columnInsert.execute();
+		columnData.clear();
```
Comment (vanchi-zendesk) [inline src/main/java/com/zendesk/maxwell/schema/SchemaStore.java]: nice shot.

**Comment by vanchi-zendesk** [comment]
:+1:

File: src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java (hunk 1 above)
Comment (osheroff) [inline src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java]: I'm not totally sure, but I think that for transactions that touch multiple tables you'll see a `BEGIN` at the the top of them; you have to keep things atomic from the slaves' perspective too... ? 

I don't actually know.

**Pull request merged**